examples/developer-tools/src/index.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```

Comment:
Do you wanna update the licenses to 2024? I don't think it matters but not sure if it does.